doc/intl.md

```diff
@@ -0,0 +1,192 @@
+# Overview

```

Comment:
Before this, can you add an ID and title for docusaurus?
The "id" is used in hermes/website/sidebars.json as a key, and "title" is displayed as the overarching title.

Make sure to also add to the sidebars.json file so this will be displayed on our website.
```suggestion
---
id: intl
title: Intl Library
---
# Overview
```

Reply:
maybe `title: Internationalization APIs`